src/coreComponents/physicsSolvers/SolverBase.cpp

```diff
@@ -203,20 +201,21 @@ void SolverBase::setNextDtBasedOnNewtonIter( real64 const & currentDt,
                                              real64 & nextDt )
 {
   integer & newtonIter = m_nonlinearSolverParameters.m_numNewtonIterations;
-  int const iterCutLimit = m_nonlinearSolverParameters.dtCutIterLimit();
-  int const iterIncLimit = m_nonlinearSolverParameters.dtIncIterLimit();
+  integer const iterCutLimit = m_nonlinearSolverParameters.dtCutIterLimit();
+  integer const iterIncLimit = m_nonlinearSolverParameters.dtIncIterLimit();
 
-  if( newtonIter <  iterIncLimit )
+  constexpr char const msg[] = "{}: Newton solver converged in less than {} iterations, time-step required will be {}.";
+  if( newtonIter < iterIncLimit )
   {
     // Easy convergence, let's double the time-step.
     nextDt = 2*currentDt;
-    GEOSX_LOG_LEVEL_RANK_0( 1, getName() << ": Newton solver converged in less than " << iterIncLimit << " iterations, time-step required will be doubled." );
+    GEOSX_LOG_LEVEL_RANK_0( 1, GEOSX_FMT( msg, getName(), iterIncLimit, "doubled" ) );
   }
-  else if( newtonIter >  iterCutLimit )
+  else if( newtonIter > iterCutLimit )
   {
     // Tough convergence let us make the time-step smaller!
     nextDt = currentDt/2;
-    GEOSX_LOG_LEVEL_RANK_0( 1, getName() << ": Newton solver converged in more than " << iterCutLimit << " iterations, time-step required will be halved." );
+    GEOSX_LOG_LEVEL_RANK_0( 1, GEOSX_FMT( msg, getName(), iterCutLimit, "halved" ) );
```

Comment:
I did not spot it at first. But reading your answer about factorisation, I noticed it.
Here first line says "_less_" and second says "_more_".
While you took care of "_doubled_" and "_halved_", your refactoring was a bit harsh 😉 

About the searchability of error messages, who will really search for `Newton solver converged in {} than` instead of `Newton solver converged in less than`?

Reply:
Thanks for catching this! Perhaps factoring out the common part of these messages is not worth it, since they are actually quite different (and may need to be worded independently).

Reply:
No pb! In this precise case, I think simplicity outperforms factorisation.